src/tryTable.ts

```diff
@@ -0,0 +1,49 @@
+import PDFDocument from 'pdfkit-table';
```

Comment:
I think this file is out of context. Please try running `npx nest g res pdf-conversion`. Please remove the controllers, dtos and entity files because you won't be needing this.

To make your solution elegant, you should transform this file into a service file.

You may refer to other documentation or example codes you see on the internet.